### ISSUE TITLE
UX Enhancement: When editing a label default the input to the Text field 

### ISSUE THREAD

**Issue by greenonline** [bancika/diy-layout-creator#781]
When double-clicking a label (Misc>Label), default the input to the **Text:** field, rather than the **Color:** field, as it currently does, in v 4.37.0. It would seem more likely that the user would want to edit the text rather than the colour. This would result in a better UX and less mouse pointing and clicking.

**Comment by bancika** [comment]
Good idea...in general it should default to whatever is the "value" of the component

**Comment by organbender** [comment]
I agree with greenonline
